react/features/notifications/reducer.ts

```diff
@@ -112,7 +118,7 @@ function _insertNotificationByPriority(notifications: INotification[], notificat
     }
 
     // Create a copy to avoid mutation and insert the notification.
-    const copyOfNotifications = notifications.slice();
+    const copyOfNotifications = filteredNotifications.slice();
```

Comment:
This is no longer necessary since the `filter()` operation will have created a new array.

Reply:
Correct. Will update.